src/AutoRest.CSharp/Mgmt/Decorator/Transformer/SchemaFormatByNameTransformer.cs

```diff
@@ -43,7 +43,7 @@ internal static void Update()
         private IEnumerable<Schema> allGeneralSchemas;
         private IEnumerable<OperationGroup> allOperationGroups;
         private IReadOnlyDictionary<string, string> allFormatByNameRules;
-        private Dictionary<Guid, string> schemaCache = new Dictionary<Guid, string>();
+        private Dictionary<Schema, string> schemaCache = new();
```

Comment:
Not sure here the `Schema` as key would drop down the perf

Reply:
Absolutely no. We are doing that a lot (in MgmtOutputLibrary). These factors impact performance very minorly